src/topology/algebra/ordered.lean

```diff
@@ -738,13 +890,11 @@ lemma not_tendsto_at_bot_of_tendsto_nhds [no_bot_order α]
 hf.not_tendsto (disjoint_nhds_at_bot x)
 
 /-!
-### Neighborhoods to the left and to the right
+### Neighborhoods to the left and to the right on an order_topology
 
-Limits to the left and to the right of real functions are defined in terms of neighborhoods to
-the left and to the right, either open or closed, i.e., members of `nhds_within a (Ioi a)` and
-`nhds_wihin a (Ici a)` on the right, and similarly on the left. Such neighborhoods can be
-characterized as the sets containing suitable intervals to the right or to the left of `a`.
-We give now these characterizations. -/
+We've seen some properties of left and right neighborhood of a point in an order_closed_topology.
+In an order_topology, such neighborhoods can be characterized as the sets containing suitable
+intervals to the right or to the left of `a`. We give now these characterizations. -/
```

Comment:
Maybe backticks around Lean terms in the comments make them easier to read? (My copy-editor instinct here wondered why you had an underscore in the title here...)

Reply:
Indeed !